atlasdb-commons/src/main/java/com/palantir/util/AssertUtils.java

```diff
@@ -119,7 +120,11 @@ public static void assertAndLogWithException(boolean cheapTest, String msg, Thro
     public static void assertAndLogWithException(Logger log, boolean cheapTest, String format, Throwable t,
             Object... args) {
         if (!cheapTest) {
-            assertAndLogWithException(log, false, String.format(format, args), t);
+            Object[] newArgs = new Object[args.length + 1];
+            newArgs[0] = t;
```

Comment:
Have you verified this works as expected? I think the throwable needs to be the last parameter.